src/json/text.json

```diff
@@ -808,6 +808,9 @@
 	"popupSettingsOnboardingModeTitle": "Network",
 	"popupSettingsOnboardingNetworkTitle": "Self-hosted Configuration",
 	"popupSettingsOnboardingStoragePath": "Account Data Location",
+	"popupSettingsOnboardingLocalOnlyWarningTitle": "Are you sure?",
```

Comment:
This is very common string, change it to commonAreYouSure and replace all existing cases with it.